# .yaml config file
#
# The PID gains and controller settings must be saved in a yaml file that gets loaded
# to the param server via the roslaunch file (my_robot_control.launch).

robomuse:
  # Publish all joint states -----------------------------------
  # Creates the /joint_states topic necessary in ROS
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Effort Controllers ---------------------------------------
  leftWheel_effort_controller:
    type: effort_controllers/JointEffortController
    joint: chassis_to_left_wheel
    pid: {p: 6, i: 0.1, d: 0.0}
    
  rightWheel_effort_controller:
    type: effort_controllers/JointEffortController
    joint: chassis_to_right_wheel
    pid: {p: 4.5, i: 0.1, d: 0.0}
#pid: {p: 50.0, i: 0.1, d: 0.0}
